### ISSUE TITLE
Keyboard navigation doesn't work when grid is filtered

### ISSUE THREAD

**Issue by Mika83AC** [vaadin/vaadin-grid#930]
See your example [https://cdn.vaadin.com/vaadin-core-elements/master/vaadin-grid/demo/sorting-filtering.html#filtering](https://cdn.vaadin.com/vaadin-core-elements/master/vaadin-grid/demo/sorting-filtering.html#filtering)

Clicking the grid content and navigating with arrow up/down works when no filter is applied.

When a filter gets applied, clicking the grid content and trying to navigate with arrow keys doesn't work.

Regards,
Michael

**Comment by Mika83AC** [comment]
I guess it is a focus problem.

When hitting the TAB key often enough (until all filter inputs have been iterated) a cell in the grid content gets highlighted (with a cell border). Now it's possible to navigate the content with the arrow keys again.

**Comment by samiheikki** [comment]
Discussed with @tomivirkki and decided to close the ticket.
